meta-lmp-bsp/recipes-bsp/u-boot/u-boot-ostree-scr-fit/am62xx-evm/boot.cmd

```diff
@@ -3,12 +3,15 @@ setenv bootlimit 3
 setenv devtype mmc
 setenv devnum ${mmcdev}
 
+# Use "#conf-overlayname.dtbo"
```

Comment:
Is it correct? `"#conf-overlayname.dtbo"`, not `"#conf-ti_overlayname.dtbo"`

Reply:
> I think the correct template is `"#conf-ti_overlayname.dtbo"`, isn't it?

Sure, updated to follow that suggestion.

Reply:
> The problem there is that it will fail when empty, and the reason why the variable has #conf in it.


Ah, the `dtoverlay` is optional. Sure, you're right. Thanks for the explanation!